react/src/components/molecules/HeaderSearch/index.js

```diff
@@ -62,7 +62,7 @@ HeaderSearch.propTypes = {
   /** The placeholder text for the input */
   placeholder: PropTypes.string,
   /** The Search button */
-  buttonSearch: PropTypes.shape(ButtonSearch.propTypes),
+  buttonSearch: PropTypes.shape(ButtonWithIcon.propTypes),
```

Comment:
This prop is no longer being used in render. I don't believe that's intentional?

Reply:
buttonSearch is accessed on line 49

Reply:
I forgot to add the props before...